src/components/Text/src/Text.vue

```diff
@@ -169,7 +207,15 @@ export default {
 </script>
 
 <style module="$s">
-.Paragraph {
+.Text {
+	margin: 0;
+	color: inherit;
+	font-weight: inherit;
+	font-family: inherit;
+	font-style: inherit;
+	text-align: inherit;
+	text-transform: inherit;
```

Comment:
Setting the margin to zero may be unexpected. Since we're agnostic to a normalize existing -- the expectation here is that additional external style could be styling text output, particularly with margins. But changing this to margin: 0 now could create unintended consequences.

Also why do all these need to be explicitly defined with inherit?

Reply:
I imported this style from MHeading, [which has had `margin: 0;` since 2020, although the earliest commit in this repo is from jan 2021](https://github.com/square/maker/blob/458fd80b45541f6ba4598cecb10f7b213f20cf4a/src/components/Heading/src/Heading.vue#L110).

I tested the behavior trying out all of the different tags and spans and divs don't have any margin by default, neither do lis, and we explicitly and intentionally removed it from h1s - h6s, so it makes ps the only exception. While this is technically a breaking change, I think it makes the behavior of all of the different element tags a user can select on a MText more consistent (i.e. none of them provide their own margin) which is better.

I've removed the explicit inherits.

Reply:
Removing the margin on default paragraphs does have an impact, it's unexpected and I think it should and can be avoided.

Reply:
FWIW, Website's renderer reset.css currently resets all `p` (among other text tags) to `margin: 0`. I think as long as we are not inlining the `margin: 0` style it should be fine. MHeading currently applies class-based margins from scoped styles over the Maker `margin: 0` style.